cmd/kubelet/app/server.go

```diff
@@ -220,6 +214,12 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 				}
 			}
 
+			// validate the initial KubeletFlags
```

Comment:
Right now this breaks precedence (see https://github.com/kubernetes/kubernetes/issues/56171) - Not sure whether re-ordering loading or changing the validation process is likely to be easier here tbh.

Might be worth looking into how `if err := kubeletConfigFlagPrecedence(kubeletConfig, args); err != nil {` works